### PR TITLE
Symlink local-bootstrap yodeploy config

### PR DESCRIPTION
**Pull request by stefanor**
For easier future updates.

Also, stop asking annoying questions.

### CODE REVIEW

File: scripts/local-deploy-bootstrap.py [hunk 1]
```diff
@@ -79,6 +79,32 @@ def binary_available(name):
...
+        with open(fn) as f:
+            for line in f:
+                if line.strip() in ('. "$HOME/.bashrc"', '. ~/.bashrc'):
+                    return
+
+        create_profile = fn
+        break
+    else:
+        if confirm("You don't seem to have a .profile.\n"
+                   "Create it, sourcing .bashrc?"):
```
Comment (snitch) [inline scripts/local-deploy-bootstrap.py]: Q000 Remove bad quotes.

Comment (stefanor) [inline scripts/local-deploy-bootstrap.py]: Remove bad @snitch.

File: scripts/local-deploy-bootstrap.py [hunk 2]
```diff
@@ -79,6 +79,32 @@ def binary_available(name):
...
+    create_profile = None
+
+    for profile in ('~/.bash_profile', '~/.bash_login', '~/.profile'):
+        fn = os.path.expanduser(profile)
+        if not os.path.isfile(fn):
+            continue
+
+        with open(fn) as f:
+            for line in f:
+                if line.strip() in ('. "$HOME/.bashrc"', '. ~/.bashrc'):
```
Comment (beck) [inline scripts/local-deploy-bootstrap.py]: Personal pref, I like using `source` over `.`
Can you add checks for that too?

Comment (stefanor) [inline scripts/local-deploy-bootstrap.py]: Time for a regex, I think.

xkcd 208

File: scripts/local-deploy-bootstrap.py [hunk 3]
```diff
@@ -79,6 +79,33 @@ def binary_available(name):
...
+            for line in f:
+                if re.match(r'^\s*(\.|source)\s+"?(~|$HOME)/\.bashrc"?\s*$',
+                            line):
+                    return
+
+        create_profile = fn
+        break
+    else:
+        if confirm("You don't seem to have a .profile.\n"
+                   "Create it, sourcing .bashrc?"):
```
Comment (snitch) [inline scripts/local-deploy-bootstrap.py]: Q000 Remove bad quotes.

File: scripts/local-deploy-bootstrap.py [hunk 4]
```diff
@@ -74,11 +74,39 @@ def abort(message):
...
```
Comment (snitch) [inline scripts/local-deploy-bootstrap.py]: Q000 Remove bad quotes.

File: scripts/local-deploy-bootstrap.py [hunk 5]
```diff
@@ -87,29 +115,18 @@ def check_environment():
...
-                 "Create it, sourcing .bashrc?"):
-        profile = os.path.expanduser('~/.profile')
-    if profile:
-        with open(profile, 'a') as f:
-            f.write("# Created by Yola's local-bootstrap\n. ~/.bashrc\n")
+    if not binary_available('build-virtualenv'):
+        bin_ = os.path.expanduser('~/bin')
+        if not os.path.isdir(bin_):
+            if confirm("You don't seem to have a ~/bin directory.\n"
+                       "Create it, and add a line to your .bashrc to add it "
```
Comment (snitch) [inline scripts/local-deploy-bootstrap.py]: Q000 Remove bad quotes.

File: scripts/local-deploy-bootstrap.py [hunk 6]
```diff
@@ -87,29 +115,18 @@ def check_environment():
...
+                       "to your PATH?"):
```
Comment (snitch) [inline scripts/local-deploy-bootstrap.py]: Q000 Remove bad quotes.

File: scripts/local-deploy-bootstrap.py (hunk 1 above)
Comment (beck) [inline scripts/local-deploy-bootstrap.py]: I don't understand why it thinks these are bad?

**Pull request merged**